### PR TITLE
[Admin] Product Attributes View - Configurations

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
|-----------------|-----
| Branch?         | 2.0 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | continuation of https://github.com/Sylius/Sylius/pull/16412
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Page/Admin/Crud/CreatePage.php [hunk 1]
```diff
@@ -39,6 +39,8 @@ public function create(): void
             $this->getDocument()->find('css', 'body')->click();
         }
         $this->getDocument()->pressButton('Create');
+
+        $this->waitForFormUpdate();
```
Comment (GSadee) [inline src/Sylius/Behat/Page/Admin/Crud/CreatePage.php]: Is it necessary?

File: src/Sylius/Behat/Page/Admin/ProductAttribute/CreatePage.php [hunk 2]
```diff
@@ -13,81 +13,8 @@
 
 namespace Sylius\Behat\Page\Admin\ProductAttribute;
 
-use Behat\Mink\Exception\ElementNotFoundException;
-use Sylius\Behat\Behaviour\SpecifiesItsField;
 use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
 
 class CreatePage extends BaseCreatePage implements CreatePageInterface
```
Comment (GSadee) [inline src/Sylius/Behat/Page/Admin/ProductAttribute/CreatePage.php]: Remove the entire class if it is empty now

File: src/Sylius/Bundle/AdminBundle/templates/product_attribute/form/configuration.html.twig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% set form = hookable_metadata.context.form %}
+
+{% if form.configuration is defined %}
+    {% set attribute_type = hookable_metadata.context.form.vars.value.type %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/product_attribute/form/configuration.html.twig]:
```suggestion
    {% set attribute_type = form.vars.value.type %}
```

**Review by GSadee — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/templates/product_attribute/form/configuration.html.twig [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
+    <div class="card g-col-12 g-col-xl-12" {{ sylius_test_html_attribute('configuration') }}>
+        <div class="card-header">
+            <div class="card-title">
+                {{ 'sylius.ui.configuration'|trans }}
             </div>
         </div>
+        <div class="card-body row">
+            {% hook 'configuration' %}
+            {% hook 'configuration' ~ '.' ~ attribute_type %}
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/AdminBundle/templates/product_attribute/form/configuration.html.twig]:
```suggestion
            {% hook 'configuration.' ~ attribute_type %}
```

File: src/Sylius/Bundle/AdminBundle/templates/shared/helper/translations.html.twig [hunk 5]
```diff
@@ -30,9 +30,9 @@
         {% for locale, translationForm in translationFields %}
             <div class="accordion-item">
                 {{ accordion_header(locale, accordion_id, loop.first) }}
-                <div id="translation-{{ locale }}" class="accordion-collapse collapse {% if loop.first %}show{% endif %}" data-bs-parent="#{{ accordion_id }}">
+                <div id="translation-{{ accordion_id }}-{{ locale }}" class="accordion-collapse collapse {% if loop.first %}show{% endif %}" data-bs-parent="#{{ accordion_id }}">
                     <div class="accordion-body row">
-                        {% hook hook_name|default('translations') with { form: translationForm, locale, _prefixes: prefixes } %}
+                        {% hook accordion_id with { form: translationForm, locale, _prefixes: prefixes } %}
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/AdminBundle/templates/shared/helper/translations.html.twig]: Why?

Comment (Rafikooo) [reply src/Sylius/Bundle/AdminBundle/templates/shared/helper/translations.html.twig]: A mistake

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @Rafikooo!

**Pull request merged**